README.md

```diff
@@ -36,14 +36,18 @@ changes [here](https://github.com/n8n-io/n8n/blob/master/packages/cli/BREAKING-C
   [how to run n8n in **Docker**](https://docs.n8n.io/hosting/installation/docker/)
 
 ## Start
+You can try n8n without installing it using npx. You must have [Node.js](https://nodejs.org/en/) installed. 
+From the terminal, run:
 
-Execute: `npx n8n`
+`npx n8n`
+
+This command will download everything that is needed to start n8n. You can then access n8n and start building workflows by opening [http://localhost:5678](http://localhost:5678).
 
 ## n8n cloud
 
-Sign-up for an [n8n cloud](https://www.n8n.io/cloud/) account.
+Sign-up for a [n8n cloud](https://www.n8n.io/cloud/) account.
```

Comment:
since the IPA pronunciation of `n8n` is `ɛn eɪt ɛn`, `an` is the correct article here.

Reply:
Okay, thanks! Didn't know that

```suggestion
Sign-up for an [n8n cloud](https://www.n8n.io/cloud/) account.
```